### PR TITLE
Fix assorted IE layout/JS issues

### PR DESCRIPTION
**Pull request by davidpmccormick**
## What is this PR trying to achieve?

Fixing a handful of IE layout issues and some JS errors.

### CODE REVIEW

File: client/js/util.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ const KEYS = {
 };
 
 const nodeList = (nl) => {
-  return [...nl];
+  return Array.prototype.slice.call(nl || []);
```
Comment (davidpmccormick) [inline client/js/util.js]: Babel's internal `_toConsumableArray` method was breaking in IE10 even though we're using the es2015 preset. I couldn't figure out why so reverted to the old-skool/working way.

Comment (jamesgorrie) [inline client/js/util.js]: Could we not use babel-polyfill?

Comment (davidpmccormick) [inline client/js/util.js]: > Could we not use babel-polyfill?

I'm not familiar with it – I had thought that the es2015 preset we're using was the magic required to make our code compatible with es2015 browsers (obviously not I guess).

We should definitely use something that makes this happen if we can.

Comment (jamesgorrie) [inline client/js/util.js]: [Here's the docs](https://babeljs.io/docs/usage/polyfill/).

Comment (davidpmccormick) [inline client/js/util.js]: Now using babel-polyfill.

**Pull request merged**